packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -702,11 +338,49 @@ async function savePage(
       'HTMLComplete'
     );
     return true;
-  } catch {
+  }
+  catch(err) {
     return false;
   }
 }
 
+async function beforeTests(unlockKeychain=false) {
+  let keychain;
+  if (unlockKeychain) {
```

Comment:
A test failed at probably just the wrong spot and it looks like it trashed my keychain. Couldn't figure out how to get it back, ended up having to remove it and restart and log back in to a lot of things, regenerate my personal access token, etc... So figured maybe I'll only run this code for test files that really need it and then I can at least focus my own test file that doesn't need it for now until I can get to the bottom of it.

Reply:
For the future reference you can use `security` command to reattach it back, it's not going anywhere